models/attachment.go

```diff
@@ -1,11 +1,149 @@
 package models
 
+import (
+	"archive/zip"
+	"bytes"
+	"encoding/base64"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
 // Attachment contains the fields and methods for
 // an email attachment
 type Attachment struct {
-	Id         int64  `json:"-"`
-	TemplateId int64  `json:"-"`
-	Content    string `json:"content"`
-	Type       string `json:"type"`
-	Name       string `json:"name"`
+	Id          int64  `json:"-"`
+	TemplateId  int64  `json:"-"`
+	Content     string `json:"content"`
+	Type        string `json:"type"`
+	Name        string `json:"name"`
+	vanillaFile bool   // Vanilla file has no template variables
```

Comment:
Could we think up a different name than `vanillaFile`? It's not really clear what this stands for (naming is hard!)

Maybe something like `isTemplate`?

Reply:
Yeah I was struggling on the name. We need something to indicate '_not a template file_' as we use the default condition of false in the logic, so it'd need to be `isNotTemplate`, which seems a bit negative. Maybe `isPlain` or `withoutTemplate`?